### DISCUSSION TITLE
[08/29] 월말 발표 내용 선정

### DISCUSSION THREAD

**Discussion by aicompose** [CodeMedics/Harulog-backend#64] [Announcements]
### css 없는 화면 기능 추가

---
* 더 넣고 싶은 내용 있으면 댓글로 추가 부탁드립니다.

**Comment by Oleevia01** [comment]
- 기획 의도
    참고: https://hong-chii.tistory.com/112

- 9월 프로젝트 개발 계획 및 일정

- 테스트 코드 내용 및 결과 (엑셀 정리)
    참고 : https://brunch.co.kr/@junovoir/29

**Comment by aicompose** [comment]
발표내용, 발표자, 내용을 구체적으로 나눌건지

발표 내용 - 
1. 기획 의도 
2. 코딩 컨벤션 -  review, 3erge 에 대한 재정립 , 주마다 사다리 등으로 짝 정하기
3. 앞으로의 계획 - 추가되는 기능
4. 월 단위 계획 -  
5. 코드 리뷰 - 문제 해결, 깃액션, jwt, 예외처리
6. 어떤 식으로 고칠 것인지 고민하고 있는 흔적 - crud 에서 발생한 문제, test 코드에서 발생한 문제에 대한 해결
